internal/test/issues/issue-1378/bionicle/generate.go

```diff
@@ -0,0 +1,3 @@
+package bionicle
+
+//go:generate go run github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen --config=config.yaml ../bionicle.yaml
```

Comment:
Will this work without a Go workspace set up in the environment? Is `go generate` smart enough to know this is the same repo, and run the local version?

Reply:
Short answer is yep!

Via https://www.jvt.me/posts/2022/06/15/go-tools-dependency-management/ and how that works.

As this is a submodule, we need to do https://github.com/deepmap/oapi-codegen/blob/5cdb5595bd38525b0a8f19cd322b91c0fb34b4d6/internal/test/go.mod#L1-L5 too